Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -713,6 +725,7 @@ keyword :
      | DOEVENTS
      | DOUBLE
      | END
+	 | ENDPROPERTY
```

Comment:
tab/spaces?

Reply:
While we're on that: @retailcoder also introduced a tabs vs spaces mismatch in the PR to exclude certain identifiers from linelabels

Reply:
I will go through the entire file and replace all the tabs with the usual number of spaces; there are a lot of mismatches.